numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1835,6 +1835,17 @@ def add_newdoc(place, name, doc):
     >>> np.left_shift(5, [1,2,3])
     array([10, 20, 40])
 
+    Note: The dtype of the second argument may change the dtype of the first
+    argument (see :ref:`Casting Rules <ufuncs.casting>`) and lead to unexpected
```

Comment:
```suggestion
    Note that the dtype of the second argument may change the dtype of the
    result and can lead to unexpected results in some cases
    (see :ref:`Casting Rules <ufuncs.casting>`):
```
Thanks for following up! Some thoughts only, but I like this either way. The only real suggestion is to change it to the `dtype of the result` (or output) instead of `first argument`.

Reply:
That sounds good.  Thanks!  I made the changes you suggested.